### ISSUE TITLE
Documentation of methods within module does not appear

### ISSUE THREAD

**Issue by dsuckau** [jsdoc3/jsdoc#1281]
Hey folks,

I have the following code:

```
/**
 * @module core/App
 */

/**
 * Description of the class.
 *
 * @class App
 * @memberof module:core/App
 */
export default class App {

    /**
     * Description of the method.
     *
     * @param {String} namespace Dot separated namespace
     * @param {*} [value=null] Value
     * @return {App}
     */
    ns(namespace, value = null) {
        // ..
    }

}
```

This way the documentation of the method `ns()` will not appear in the generated HTML. I tried different combinations with @memberof, @lends and @function, but nothing helped.

What point am I missing?

Greets from Germany

**Comment by liuhann** [comment]
Hi

use something like this:

      @method ns
      @memberof module:core/App.App 

so your @memberof annotation is  module:moduleName.className

**Comment by hegemonic** [comment]
This sounds like a duplicate of #1293, which is now fixed on GitHub master. The fix will be included in JSDoc 3.5.0.
